lua/nvim-treesitter/shell_command_selectors.lua

```diff
@@ -169,22 +173,38 @@ function M.select_download_commands(repo, project_name, cache_folder, revision)
       M.select_install_rm_cmd(cache_folder, project_name..'-tmp')
     }
   else
+    local git_folder = utils.join_path(cache_folder, project_name)
+    local clone_error = 'Error during download, please verify your internet connection'
+    if is_windows then
+      clone_error = clone_error .. ". If on Windows you may need to enable Developer mode"
+    end
+
     return {
       {
         cmd = 'git',
         info = 'Downloading...',
-        err = 'Error during download, please verify your internet connection',
+        err = clone_error,
         opts = {
           args = {
             'clone',
             '--single-branch',
             '--branch', repo.branch or 'master',
```

Comment:
Can you try whether removing `--single-branch` and `--branch` fixes the installation problems with Gitlab? They should not be needed when the version is selected later with the checkout.

Reply:
If possible, only try remove `--single-branch` i guess this could be the culprit why all `main` repos fail on windows CI.

Reply:
But thinking about it... the branch was only needed to enable `--depth=1`

Reply:
To be honest, I'm a little confused regarding what to do here. Especially considering the first review comment. 😉 

is `--branch` not needed anymore because `--depth=1` was deleted anyway?

Reply:
`--depth=1` was needed because we wanted to clone only the commit we need. This is much faster than doing a full-clone and then a checkout of the commit we want. But `--branch` only works with branches and not with individual commits that are not tree.

If the longer download times are a problem for you, you could use the git method only for parsers with symlinks (they could be marked as such)

Reply:
I don't care for longer download times. 😃 
Changed it right now, works for me. I'm waiting for CI and have a look.

Reply:
Yes, we want to make stuff work for users :slightly_smiling_face: . We want to get to a code that causes least problems on most machines. :tada: CI seems to work now!